jaxrs/src/main/java/se/fortnox/reactivewizard/ExceptionHandler.java

```diff
@@ -72,7 +73,8 @@ public Publisher<Void> handleException(HttpServerRequest request, HttpServerResp
             webException = new WebException(BAD_REQUEST, "invalidjson", throwable.getMessage());
         } else if (throwable instanceof WebException we) {
             webException = we;
-        } else if (throwable instanceof ClosedChannelException || throwable instanceof AbortedException || throwable instanceof CancellationException) {
+        } else if (throwable instanceof ClosedChannelException || throwable instanceof AbortedException || throwable instanceof CancellationException
+            || (throwable instanceof IOException && throwable.getMessage().contains("Broken pipe"))) {
```

Comment:
Could there be a risk of IOException having a null message? It can be done from someone/something throwing an io exception this way?

```
var io = new IOException();

assertThat(io.getMessage())
	.isNull();
```